hnn_core/network.py

```diff
@@ -650,9 +700,18 @@ def _create_drive_conns(self, target_populations, weights_by_receptor,
         synaptic_delays : dict or float
             Synaptic delay (in ms) at the column origin, dispersed laterally as
             a function of the space_constant
+        n_drive_cells : int
+            The number of drive cells (i.e., ArtificialCell objects) that
+            contribute to this drive. If n_drive_cells=self.n_cells and
+            cell_specific=True, artificial drive cells get assigned to each
```

Comment:
```suggestion
            cell_specific=True, artificial drive cells get connected to each
```

"assigned" feels like a weird word to use here

Reply:
I guess what I mean by "assigned" is that they are created and connected, not just connected. My intention was to build intuition for what the user can expect regarding the number of artificial cells that get created without having to bog them down with the details.